repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/actor.py

```diff
@@ -0,0 +1,28 @@
+import os
+import errno
+
+from leapp.actors import Actor
+from leapp.models import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+from leapp.libraries.actor.library import generate_report
+
+
+class LocalNetworkScriptsCheck(Actor):
+    """
+    Check for existence of network-scripts local scripts and warn user about possible
+    manual intervention requirements.
+    """
+
+    name = "local_network-scripts_check"
+    produces = (Report,)
+    tags = (IPUWorkflowTag, ChecksPhaseTag)
```

Comment:
The actor has to be executed in the `FactsPhaseTag` phase instead.
* in case an actor has to work with the system it cannot be part of check-phases. (imagine that actors executed in the checksphase should be possible to execute on a remote machine, so they can work just with information delivered via messages and cannot call any commands on the system, so cannot check whether particular files exists). Ideally it should be 2 actors (one getting the data and producing a msg for the check actor, and check actor that create the inhibitor), but in this case, let's rather keep it just one actor and move it to the factscollection phase.

Reply:
Thanks for the explanation, I used `FactsPhaseTag` phase instead.